erigon-lib/state/history.go

```diff
@@ -214,24 +226,33 @@ func (h *History) openDirtyFiles() error {
 		for _, item := range items {
 			fromStep, toStep := item.startTxNum/h.aggregationStep, item.endTxNum/h.aggregationStep
 			if item.decompressor == nil {
-				fPath := h.vFilePath(fromStep, toStep)
-				exists, err := dir.FileExist(fPath)
-				if err != nil {
+				fPathMask := h.vFilePathMask(fromStep, toStep)
+				fPath, fileVer, err := version.FindFilesWithVersionsByPattern(fPathMask)
+				if err != nil && !errors.Is(err, os.ErrNotExist) {
```

Comment:
i would move `errors.Is` inside `FindFilesWithVersionsByPattern` and return one more param `ok bool`